compiler/rustc_codegen_cranelift/src/main_shim.rs

```diff
@@ -85,7 +85,7 @@ pub(crate) fn maybe_create_entry_wrapper(
             let arg_argc = bcx.append_block_param(block, m.target_config().pointer_type());
             let arg_argv = bcx.append_block_param(block, m.target_config().pointer_type());
             let arg_ignore_sigpipe = bcx.append_block_param(block, types::B1);
-            // We also need to set the value of `arg_ignore_sigpipe` to `ignore_sigpipe`
+            // How? arg_ignore_sigpipe.value = _ignore_sigpipe;
```

Comment:
You can use `let arg_ignore_sigpipe = bcx.ins().iconst(types::I8, ignore_sigpipe as i64);`